.circleci/config.yml

```diff
@@ -14,6 +14,9 @@ jobs:
           key: dependency-cache-2-{{ checksum "package-lock.json" }}
           paths:
             - ./node_modules
+      - run:
+          name: Build
+          command: npm start & sleep 120; npm stop
```

Comment:
We talked about this briefly on Slack, but I think it might be better to just run `npm run build-production`, which is not a never-ending process. The thing I was most concerned about in #287 was really just that the kind of build we do when releasing works (i.e. `npm run build-production`), and not so much that the dev server functions.

If you *also* want to check that the dev server starts successfully, a nicer approach might be:

```sh
# Start the dev server in the background
npm start &

# Get its process ID
DEV_SERVER_PID=$!

# Wait a bit for it to initialize (120s seems longer than necessary
# to me, and really slows our tests down)
sleep 10

# Now stop the process! The return code of `kill` will be 0 if the
# process was running and 1 if not (in which case, we should error
# anyway because the server must have crashed/failed to start).
#
# Circle sets `-e` on bash, which means it will exit with an error
# as soon as any command returns a non-zero exit code, so we don't
# need to do anything special except try to kill the process. If
# Killing fails, we should fail :)
kill $DEV_SERVER_PID
```

…this way, you don’t need any of the extras you added to the server or the `package.json` scripts.

Reply:
Oops...missed that comment :/ Will update my PR to just build application, thanks!